__fixtures__/test-project/web/tsconfig.json

```diff
@@ -3,9 +3,9 @@
     "noEmit": true,
     "allowJs": true,
     "esModuleInterop": true,
-    "target": "esnext",
-    "module": "esnext",
-    "moduleResolution": "node",
+    "target": "ES2023",
```

Comment:
This to me is the only one worth being _"are you sure"_ on, you're making the assumption of a very modern browser at the type-level, potentially without a polyfill?

Reply:
Good point! I'm guessing you're happy with moving away from next to at least to some pinned version? Just a question of how far back - do you have a preference? 

Looking through caniuse for es2023 it has support in the high 80s which might not be enough.

Users are free to change this but how likely it is that an average user would change it is questionable.

Reply:
Yeah, switching from esnext is good

I've always thought 2 years is a healthy amount of time, so  either 2022 or 2021?

Reply:
Awesome. I've switched it to 2022 since we're nearing the end of 2024 and it so happens to look nice beside the resolution option which happens to be 2022 haha.